lib/i18n/activities.ts

```diff
@@ -540,6 +540,10 @@ export const ActivityDescriptionI18n = defineMessages({
     defaultMessage: 'Vendor <Vendor></Vendor> created',
     id: 'IUlgDG',
   },
+  ADDED_FUNDS_EDITED: {
+    defaultMessage: '<Individual></Individual> edited added funds',
```

Comment:
Maybe link to the order that was edited:

```suggestion
    defaultMessage: '<Individual></Individual> edited <Order>added funds</Order>',
```